tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/polyparser/Training.scala

```diff
@@ -57,29 +63,47 @@ object Training {
             }
           }
     }
-    val config: ParserTrainingConfig = optionParser.parse(args, ParserTrainingConfig()).get
+    val trainingConfig: ParserTrainingConfig = optionParser.parse(args, ParserTrainingConfig()).get
     val trainingSource: PolytreeParseSource =
-      MultiPolytreeParseSource(config.trainingPath.split(",") map { path =>
+      MultiPolytreeParseSource(trainingConfig.trainingPath.split(",") map { path =>
         InMemoryPolytreeParseSource.getParseSource(
           path,
-          ConllX(true), config.dataSource
+          ConllX(true), trainingConfig.dataSource
         )
       })
 
-    val clusters: Seq[BrownClusters] = config.clustersPath match {
+    // Read in taggers config file if specified. This will contain config info necessary to
+    // initialize the required feature taggers (currently contais only Verbnet config).
+    val taggersConfigOption =
+      trainingConfig.taggersConfigPath map (x => ConfigFactory.parseFile(new File(x)))
+
+    val clusters: Seq[BrownClusters] = trainingConfig.clustersPath match {
       case Some(clustersPath) => clustersPath.split(",") map { path =>
         BrownClusters.fromLiangFormat(path)
       }
       case _ => Seq.empty[BrownClusters]
     }
 
-    val verbnetClassMap: Map[Symbol, Set[Symbol]] = config.verbnetConfigPath match {
-      case Some(polyparserConfigPathVal) => VerbnetUtil.getVerbnetClassMap(polyparserConfigPathVal)
-      case _ => Map.empty[Symbol, Set[Symbol]]
-    }
-    val taggers: Seq[SentenceTransform] =
+    var taggers: Seq[SentenceTransform] =
       Seq(FactorieSentenceTagger, LexicalPropertiesTagger,
         BrownClustersTagger(clusters))
+
+    for {
+      taggersConfig <- taggersConfigOption
+      verbnetConfig <- taggersConfig.get[Config]("verbnet")
+      groupName <- verbnetConfig.get[String]("group")
+      artifactName <- verbnetConfig.get[String]("name")
+      version <- verbnetConfig.get[Int]("version")
+    } {
+      val verbnetPath: java.nio.file.Path = Datastore.directoryPath(
+        groupName,
+        artifactName,
+        version
+      )
+      val verbnet = new Verbnet(verbnetPath)
+      taggers :+= VerbnetTagger(verbnet)
+    }
+
```

Comment:
I looked this up today and read somewhere that `Option` is like a collection, so just doing a `++ maybeVerbnetTagger` works.